<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CLD-LAB0-QAA01" name="com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka" tests="4" failures="1" timestamp="16 Mar 2017 09:47:47 GMT" time="1145.505" errors="1">
  <testcase name="verifyArchivedTickets" time="438.211" classname="com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka">
    <error type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.whs.navigator.commonUtil.WebObjectUtil.stopCurrentTestExecution(WebObjectUtil.java:1821)
	at com.whs.navigator.commonUtil.WebObjectUtil.clickElement(WebObjectUtil.java:491)
	at com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka.verifyArchivedTickets(TimovTestCase_Priyanka.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- verifyArchivedTickets -->
  <testcase name="verifyCustomerAdvocateAssignedOrg" time="289.822" classname="com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka"/>
  <testcase name="verifyArchivedTicketsBreadCrumb" time="417.460" classname="com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka"/>
  <testcase name="verifyAccSettingsChallengeQuestions" time="0.012" classname="com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	The method getNav_ChallengeQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
	The method getNav_ChallengeQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory
	The method getNav_CustomQuestionField() is undefined for the type AccountPageFactory
	The method getNav_CustomAnswerField() is undefined for the type AccountPageFactory

	at com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka.verifyAccSettingsChallengeQuestions(TimovTestCase_Priyanka.java:142)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- verifyAccSettingsChallengeQuestions -->
</testsuite> <!-- com.whs.navigator.testcases.Timov.TimovTestCase_Priyanka -->
